cmd/rook/ceph/mds.go

```diff
@@ -49,11 +49,16 @@ func init() {
 }
 
 func startMDS(cmd *cobra.Command, args []string) error {
-	required := []string{"mon-endpoints", "cluster-name", "admin-secret", "filesystem-id", "pod-name", "public-ipv4", "private-ipv4"}
+	required := []string{"mon-endpoints", "cluster-name", "admin-secret", "filesystem-id", "pod-name"}
 	if err := flags.VerifyRequiredFlags(mdsCmd, required); err != nil {
 		return err
 	}
 
+	renamed := [][2]string{{"public-ip", "public-ipv4"}, {"private-ip", "private-ipv4"}}
```

Comment:
could the set of renamed flags be defined in just one place? (perhaps the ceph root command)